ledspicer/data/animations/Pulse.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<LEDSpicer version="1.0" type="Animation">
+    <actor
+        type="PipeWire"
+        group="Player1"
+        filter="Normal"
+        mode="VuMeter"
+        direction="Forward"
+        off="DarkCyan"
+        low="Blue"
+        mid="Red"
+        high="White"
+        channel="Both"
```

Comment:
I recommend using the channel `Left` (or right depending the layout) for `player 1` and channel `Right` for `player 2`